metrics/web/web/metrics_config.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * A class representing a configuration that is used within the application.
+ */
+class MetricsConfig {
+    /**
+     * Creates a new instance of the MetricsConfig with default value for each configuration item.
+     */
+    constructor() {
+        this.sentryDsn = "$SENTRY_DSN";
+        this.sentryRelease = "$SENTRY_RELEASE";
+        this.sentryEnvironment = "$SENTRY_ENVIRONMENT";
+        this.googleSignInClientId = "$GOOGLE_SIGN_IN_CLIENT_ID";
```

Comment:
```suggestion
        this.googleSignInClientId = "$GOOGLE_SIGN_IN_CLIENT_ID";
        this.sentryDsn = "$SENTRY_DSN";
        this.sentryRelease = "$SENTRY_RELEASE";
        this.sentryEnvironment = "$SENTRY_ENVIRONMENT";
```

or maybe 
```suggestion
        /**
        * Google Sign In Configuration
        *
        * googleSignInClientId - A unique ID of the client that is used to initialize ...
        */
        this.googleSignInClientId = "$GOOGLE_SIGN_IN_CLIENT_ID";
        
        /**
        * Sentry Configuration
        *
        * sentryDsn - is Data Source Name ....
        */
        this.sentryDsn = "$SENTRY_DSN";
        this.sentryRelease = "$SENTRY_RELEASE";
        this.sentryEnvironment = "$SENTRY_ENVIRONMENT";
```

basically, maybe copy the description from `MetricsConfig` as this might be useful to users, what do you think?